shopfloor_batch_automatic_creation/actions/picking_batch_auto_create.py

```diff
@@ -111,6 +112,11 @@ def gt(value1, value2, digits):
         total_weight = 0.0
         total_volume = 0.0
         for picking in pickings:
+            if picking == first_picking:
+                # Always take the first picking even if it doesn't fullfill the
+                # weight/volume criteria. This allows to always process at least
+                # one picking which won't be processed otherwise
+                selected_pickings |= first_picking
```

Comment:
how does this behave if max_pickings is passed? Wouldn't you miss one picking in certain cases?

Reply:
Hum indeed, we should check `max_pickings` here as well and get out of the loop if needed

Reply:
@simahawk I changed a bit the implementation